src/System.Runtime.InteropServices.RuntimeInformation/tests/CheckPlatformTests.cs

```diff
@@ -5,20 +5,10 @@ namespace System.Runtime.InteropServices.RuntimeInformationTests
 {
     public class CheckPlatformTests
     {
-        [Fact, PlatformSpecific(PlatformID.Windows)]
-        public void CheckWindows()
-        {
-            Assert.True(RuntimeInformation.IsOSPlatform(OSPlatform.Windows));
-            Assert.True(RuntimeInformation.IsOSPlatform(OSPlatform.Create("WINDOWS")));
-            Assert.False(RuntimeInformation.IsOSPlatform(OSPlatform.Create("windows")));
-            Assert.False(RuntimeInformation.IsOSPlatform(OSPlatform.Create("Windows NT")));
-            Assert.False(RuntimeInformation.IsOSPlatform(OSPlatform.Linux));
-            Assert.False(RuntimeInformation.IsOSPlatform(OSPlatform.OSX));
-        }
-
         [Fact, PlatformSpecific(PlatformID.Linux)]
         public void CheckLinux()
         {
+            Assert.False(RuntimeInformation.IsOSPlatform(OSPlatform.FreeBSD));
```

Comment:
> Now the falsey assertions follow the alphabetic order. Is this correct?

I don't have a preference on that one, other than the related variations of the same platform should probably be grouped together, e.g. the false tests for "Darwin", "osx", "mac", etc. should all be grouped together.

> should we have an extra linefeed separating true and false assertions?

Sure.